core/domain/email_manager.py

```diff
@@ -1571,7 +1571,6 @@ def send_mail_to_notify_contributor_dashboard_reviewers(
         return
 
     if not reviewer_ids:
-        log_new_error('No Contributor Dashboard reviewers to notify.')
```

Comment:
Instead of dropping this logging statement, could we instead make it such that this function isn't called unless there are folks to notify? I think the logging statement is valid, and the error is that this function is being called when it shouldn't be.